cmd/libvirt-provider/app/app.go

```diff
@@ -156,6 +158,8 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.DurationVar(&o.MachineEventStore.MachineEventTTL, "machine-event-ttl", 5*time.Minute, "Time to live for machine events.")
 	fs.DurationVar(&o.MachineEventStore.MachineEventResyncInterval, "machine-event-resync-interval", 1*time.Minute, "Interval for resynchronizing the machine events.")
 
+	fs.StringVar(&o.VolumeCachePolicy, "volume-cache-policy", "none", "Policy to use when creating a remote disk. Default is none")
```

Comment:
what are other options?

Reply:
Based on my understanding, the accepted values are `none`, `writeback`, and `writethrough`. I assumed that users setting this option explicitly would already be familiar with these values, so I didn’t add any additional validations or detailed information. Additionally, if more values are supported in the future, we would need to update the validation logic, which could introduce unnecessary maintenance overhead.

Reply:
validation isn't such important. but pls add minimally possible option what can be problematic because it depends on hypervisor and libvirtd version. Maybe you can add this note and use official docs: https://libvirt.org/formatdomain.html#hard-drives-floppy-disks-cdroms here you have to scroll on attribute drive where are all options.

Reply:
Added